src/Builder/ClassModifier/ClassInstanceModifierStrategy/Applicable/ApplicableReturnChainedCollectionModifierStrategy.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Integrator\Builder\ClassModifier\ClassInstanceModifierStrategy\Applicable;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Expression;
+use PhpParser\Node\Stmt\Return_;
+use PhpParser\NodeFinder;
+
+class ApplicableReturnChainedCollectionModifierStrategy implements ApplicableInterface
```

Comment:
If we rename interface to `ClassInstanceModifierStrategyInterface`, in this case the class name will be `ReturnChainedCollectionClassInstanceModifierStrategy`. What do you think about it?

Reply:
Renamed to `ReturnChainedCollectionApplicableModifierStrategy`
By pattern `{Prefix}{implementedInterface}`